CONTRIBUTING.md

```diff
@@ -1,7 +1,21 @@
 # Contributing to the CLI
-### Setting up your environment
-* Make sure you are using go1.8 (`go version`).
+
+- [Development Setup](#development-setup)
+  * [Environment](#environment)
+  * [Building](#building)
+  * [Adding/updating new dependencies](#adding-updating-new-dependencies)
+  * [Generating mocks/licenses](#generating-mocks-licenses)
+  * [Cross-compiling](#cross-compiling)
+  * [Testing](#testing)
+- [Contributing code](#contributing-code)
+- [Amazon Open Source Code of Conduct](#amazon-open-source-code-of-conduct)
+- [Licensing](#licensing)
+
+## Development Setup
+### Environment
+* Make sure you are using go1.10 (`go version`).
```

Comment:
"Go 1.10"

Also, the [Makefile](https://github.com/aws/amazon-ecs-cli/blob/v1.4.2/Makefile#L56) is currently using Go 1.8.

Reply:
Good catch. Will fix.